benchmarks/bench_mhc_post.py

```
@@ -0,0 +1,90 @@
+    device = torch.device("cuda")
+    header = f"{'N':>5} {'H':>5} {'HC':>3} {'median us':>12} {'mean us':>12}"
+    print(header)
+    print("-" * len(header))
+    for hidden_size in args.hidden_sizes:
+        for tokens in args.sequence_lengths:
+            torch.manual_seed(0)
+            inputs = _make_inputs(tokens, hidden_size, args.hc, device)
+
+            def call() -> None:
+                flashinfer.mhc.mhc_post(
+                    inputs["x"],
```

Ruff (0.15.13)

[warning] Function definition does not bind loop variable `inputs`

(B023)